assets/css/v2/text.scss

```
@@ -33,6 +33,14 @@
     margin-bottom: 40px;
 }
 
+.text__screen-reader-link {
```

A small nit-pick: BEM naming would it make it seem like this class refers to some component nested inside of an element called `text`. (The double underscores denote a child element).

Since this is just a standalone class that doesn't depend on its parent, we don't need a nested name.

```suggestion
.screen-reader-link-text {
```